build_deploy.sh

```diff
@@ -7,16 +7,15 @@ CICD_TOOLS_URL="https://raw.githubusercontent.com/RedHatInsights/cicd-tools/main
 source <(curl -sSL "$CICD_TOOLS_URL") image_builder
 
 export CICD_IMAGE_BUILDER_IMAGE_NAME='quay.io/cloudservices/compliance-backend'
-export CICD_IMAGE_BUILDER_BUILD_ARGS=("IMAGE_TAG=$(cicd::image_builder::get_image_tag)")
+
```

Comment:
This change would also affect the non-sc branch 😞 did you check that if it would not cause any side-effects?

Reply:
```
export CICD_IMAGE_BUILDER_BUILD_ARGS=("IMAGE_TAG=$(cicd::image_builder::get_image_tag)")
````
This line is in the `else` statement. If the branch is not `"origin/security-compliance"` the execution will be the same as before.